public/javascripts/ng-apps.js

```diff
@@ -41,6 +41,23 @@ springInitiative.config(function($stateProvider, $urlRouterProvider, $locationPr
         'content': {
           templateUrl: 'views/content/student.html',
           controller: 'studentController'
+        },
+        'studentView@index.student': {
+          templateUrl: 'views/content/studentView.html'
+        }
+      }
+    })
+    .state('index.student.addEntry', {
```

Comment:
you can just do student.addEntry instead - what I meant when talking about naming conventions before is that if you have viewA viewB and viewC nested, then calling viewB would be viewA.viewB and calling viewC would be viewB.viewC, and so on.